core/common/src/main/java/com/google/jetpackcamera/core/common/CommonModule.kt

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (C) 2023 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.jetpackcamera.core.common
+
+import dagger.Module
+import dagger.Provides
+import dagger.hilt.InstallIn
+import dagger.hilt.components.SingletonComponent
+import kotlinx.coroutines.CoroutineDispatcher
+
+
+/**
+ * Dagger [Module] for Common dependencies.
+ */
+@Module
+@InstallIn(SingletonComponent::class)
+class CommonModule {
+
+    @Provides
+    fun provideDefaultDispatcher() : CoroutineDispatcher = kotlinx.coroutines.Dispatchers.Default
```

Comment:
nit: Should we provide a default scope rather than default dispatcher? This seems to give more flexibility in tests. See https://developer.android.com/kotlin/coroutines/test#inject-scope.

Reply:
I'll change this, I think I saw a guideline to provide the default dispatcher somewhere, but I was probably mistaken.

Reply:
Tracking in b/282019672